### ISSUE TITLE
Click on the button to prepare for landing verification, but did not appear landing interface, no response

### ISSUE THREAD

**Issue by tanyaoh** [dropbox/dropbox-sdk-obj-c#191]
[DropboxClientsManager authorizeFromController:[UIApplication sharedApplication] controller:self openURL:^(NSURL *url) {
                    [AppViewDelegate openURL:url options:nil];
                } browserAuth:NO];

The returned URL is:  @"dbapi-2://1/connect? k=40l8o30x6jpwf8z&s=&state=oauth2:1BA467D3-65B0-4A14-B8FE-DEDFF7109DB3";

Thanks！

**Comment by greg-db** [comment]
I'm afraid I don't understand your post. I'll be happy to help with any issues you're having with the Dropbox API, but I'll need some more information. Please reply with:

- the version number of the SDK you're using. (If you're not using the latest version, please upgrade and try again.)
- the full text of any error or output.
- the steps to reproduce the issue, including any other relevant code snippet(s).
- screenshots or a screencast showing the issue.

Thanks in advance!

**Comment by tanyaoh** [comment]
My App is using the Dropbox version of 3.3.4。Dropbox has been in normal use in our App, but there are a small number of customers in foreign countries to reflect that they can’t jump to login Dropbox page. So I can‘t provide any print information.
But I can provide some of the relevant code：

[Code_and_video 2.zip](https://github.com/dropbox/dropbox-sdk-obj-c/files/1429825/Code_and_video.2.zip)

thank you！

**Comment by greg-db** [comment]
Thanks! So, to make sure I understand, based on the supplied video, it looks like the issue is that the Dropbox app authorization flow is not shown when the "Dropbox Accessibility" row is being selected. Is that correct?

Are you able to reproduce this yourself, or is this video just from a user?

If you can reproduce this yourself, please use the debugger to determine where this is breaking down. For example:
- Is `didSelectRowAtIndexPath` being triggered?
- Is the `indexPath.row == 2` conditional being evaluated to true?

The above should help determine if `authorizeFromController` is actually being called. If it is being called, but isn't showing the app authorization flow, I would expect some error in the log. In that case, please check the error log for any information.

**Comment by tanyaoh** [comment]
Yes, the user reflects the situation is the case, select "Dropbox Accessibility" line, there is no response, that video is provided by the customer, I can not reproduce, I use them are normal and select "Dropbox Accessibility" After the line there will be Dropbox application authorization process interface appears, so I do not know how to provide error log. Do you have any advice? thank you!

**Comment by greg-db** [comment]
Does the use happen to have a computer with Xcode installed by any chance? If so, they can use the Devices manager to see/download the console log when they connect the iPhone.

Otherwise, without the ability to reproduce the issue or any error message to go on, I'm afraid I won't be able to offer much help. (E.g., from the information available so far, it's not even clear if `authorizeFromController` is actually being called to begin with.)
